deployment/kubernetes/yaml/orderhistoryservice.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ftgo-orderhistoryservice

```

Comment:
Please use hyphens in the service names and YAML file names: `order-history-service`

Reply:
updated service names and file names.